src/Raven.Studio/typescript/viewmodels/database/tasks/backups.ts

```diff
@@ -46,7 +46,7 @@ class backups extends viewModelBase {
         this.myNodeTag(this.clusterManager.localNodeTag());
         this.serverWideTasksUrl = appUrl.forServerWideTasks();
         this.ongoingTasksUrl = appUrl.forOngoingTasks(this.activeDatabase());
-        this.canNavigateToServerWideBackupTasks = accessManager.default.clusterAdminOrClusterNode;
+        this.canNavigateToServerWideBackupTasks = accessManager.default.isClusterAdminOrClusterNodeClearance;
```

Comment:
no need to change variable node - it adds extra work + makes harded to merge changes.

Reply:
done